tests/FVTTests/homeView/testHomeView.ts

```diff
@@ -201,8 +202,18 @@ describe('JES explorer function verification tests', function () {
             });
         });
         describe('Tree interaction', () => {
-            // TODO:: Implement once we have an ID for refresh icon and loading icon
-            it.skip('Should handle reloading jobs when clicking refresh icon');
+            before('Reset prior to Tree interaction suite', async () => {
+                loadPage(driver, BASE_URL);
```

Comment:
can you add await to this like this:
Not able to pass tests without these on my local
```
 await loadPage(driver, BASE_URL);
 await driver.wait(until.elementLocated(By.id('refresh-icon')));
```

Reply:
added